tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this was copied from the example repo listed in the ES6 spec. Generally I used launchpad and gearman-admin as my examples.